### DISCUSSION TITLE
Failed to start host

### DISCUSSION THREAD

**Discussion by SeananTQ** [community/community#14455] [Copilot Conversations]
**My OS : Windows 7 Service Pack 1
My IDE: Microsoft Visual Studio Community 2022  Version 17.1.3**

**I follow these steps**
[Getting Started with GitHub Copilot in Visual Studio](https://github.com/github/copilot-docs/blob/main/docs/visualstudio/gettingstarted.md#getting-started-with-github-copilot-in-visual-studio)

**However, the following error occurs in step 4:**

`GitHub Copilot 1.10.0.3 (v1.10.0.3@764d4f318)
GitHub Copilot Agent v1.10.0
[Error] An error occurred for 'LSP': StreamJsonRpc.RemoteInvocationException: Failed to start host 'node.x86$C94B8CFE-E3FD-4BAF-A941-2866DBB566FE'. Host exited with exit code '-1073741511'. (ErrorKind: Error)
   在 StreamJsonRpc.JsonRpc.<InvokeCoreAsync>d__143`1.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.ServiceHub.Client.HubClient.RemoteServiceBrokerWrapper.<>c__DisplayClass9_0.<<RequestServiceChannelAsync>b__0>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Microsoft.ServiceHub.Client.HubClient.RemoteServiceBrokerWrapper.<>c__DisplayClass9_0.<<RequestServiceChannelAsync>b__0>d.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Microsoft.ServiceHub.Utility.Shared.ServiceHubRetry.<ExecuteAsync>d__0`1.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Microsoft.ServiceHub.Utility.Shared.ServiceHubRetry.<ExecuteAsync>d__0`1.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.ServiceHub.Client.HubClient.RemoteServiceBrokerWrapper.<RequestServiceChannelAsync>d__9.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Microsoft.ServiceHub.Client.HubClient.RemoteServiceBrokerWrapper.<RequestServiceChannelAsync>d__9.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.ServiceHub.Framework.RemoteServiceBroker.<GetPipeAsync>d__32.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.ServiceHub.Client.HubClient.<GetPipeAsync>d__38.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 GitHub.Copilot.CopilotLanguageClient.<ActivateAsync>d__29.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.VisualStudio.LanguageServer.Client.RemoteLanguageClientInstance.<InitializeAsync>d__100.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Microsoft.VisualStudio.LanguageServer.Client.RemoteLanguageClientInstance.<InitializeAsync>d__100.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.VisualStudio.Threading.ThreadingTools.<WithCancellationSlow>d__9.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.VisualStudio.LanguageServer.Client.RemoteLanguageClientInstance.<ActivateLanguageClientAsync>d__107.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
RPC server exception:
Microsoft.ServiceHub.Utility.ServiceHubException: Failed to start host 'node.x86$C94B8CFE-E3FD-4BAF-A941-2866DBB566FE'. Host exited with exit code '-1073741511'. (ErrorKind: Error)
      在 Microsoft.ServiceHub.Controller.LocationService.<StartServiceAsync>d__60.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 Microsoft.ServiceHub.Controller.LocationService.<GetHostedServiceLocationAsync>d__76.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 Microsoft.ServiceHub.Controller.LocationService.<LocateAsync>d__44.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 Microsoft.ServiceHub.Controller.HubController.<LocateAsync>d__8.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 Microsoft.ServiceHub.Controller.HubController.<RequestServiceChannelAsync>d__13.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 Microsoft.ServiceHub.Controller.ClientConnection.<RequestServiceChannelAsync>d__27.MoveNext()`

**Comment by AugustoHTPereira** [comment]
Same error here.

Microsoft Visual Studio Enterprise 2022 (64 bits) Version 17.2.4

**Reply by AaronZengNZ** [reply]
same here

**Reply by shiloh92** [reply]
Same error.
